.travis.yml

```diff
@@ -1,19 +1,35 @@
 language: go
-
+sudo: false
 go: 1.4.2
-
 script:
-    # Godep's unit tests run git, and git complains
-    # if we don't set these config parameters.
-    # We put dummy values here because they don't matter.
-    - git config --global user.email "you@example.com"
-    - git config --global user.name "Your Name"
-
-    - test -z "$(go fmt)"
-    - go vet
-    - go test -v
-    - go test -v -race
-    - test -z "$(goimports -l .)"
-
+  # Godep's unit tests run git, and git complains
+  # if we don't set these config parameters.
+  # We put dummy values here because they don't matter.
+- git config --global user.email "you@example.com"
+- git config --global user.name "Your Name"
+- test -z "$(go fmt)"
+- go vet
+- go test -v
+- go test -v -race
+- test -z "$(goimports -l .)"
 before_install:
-    - go get golang.org/x/tools/cmd/vet golang.org/x/tools/cmd/goimports
+- go get golang.org/x/tools/cmd/vet golang.org/x/tools/cmd/goimports
+before_deploy:
+- export OS_TARGETS="linux darwin windows"
+- export ARCH_TARGETS="386 amd64"
+- go get github.com/mitchellh/gox
+- gox -os "$OS_TARGETS" -arch="$ARCH_TARGETS" -build-toolchain
+- gox -os "$OS_TARGETS" -arch="$ARCH_TARGETS"
+deploy:
+  skip_cleanup: true
+  provider: releases
+  api_key:
+    secure: Q1JP8LziaXMTxFmNXiyC1YhS9e4M4WnI6UDjRTMf6mm1LZeJyUFOCCtXnifL7RyCIR1hpjp6s8M1aWE+NpuweF96IZI3Uk4ASx5C8FePC4qvhsCdtJ2sLD2GTIrp9b0MS9/+ao20AIbpVDSaLaF9IjqXpMxMyM0P8P5coRTkwItlGxmQbVJW3YuiYcPa8UojwM4EyafO2CIoUKapW8lwb9KcimBJV8PfF/XZjPVhMkn2ABhh5Hqbn2zBJtvPYMMzi0CnY50JQF5LwN3vGTMpTsRP+lOLCNbOWfkl+2hgG7VpKrtx+cX62knOodpF457sIJ31KUzmeLUVBejTGb1zuVeTojuyi8Huo8YBIBCcN+p3Dqd+n2ZK45mIrheGiEJIkf/vI4MI6A01Nu/o+xU0IPsVfAL/xU5j5nntEGfFWVoclPrl9qcfqf74xdRcARzcCJVmdc8iw49DBDHJfnPa3zxzVz//00+Rz6mZXmhk+Npk/HLLNW59vmJIjP+8XOtPor7dST9HrS1a9AcnmIjNuw9yfbwK5769SDVxCKgqNwXW/Dy5F39aIH5AL4I4y9hCEeeT8ctvSJHGOyiB9MWU5jnt5tluPtz5opG51tFXnIYP/XaWpTfO+eJ6x55pbwT+n3LfRS5l1POM+jGAFF1MFWwc14RY7qynEIEzm4Wb/UE=
```

Comment:
They are encrypted using a private key associated with this repo so I believe they won't decrypt properly for another repo / fork / etc (as per Travis docs).